private/buf/bufcli/bufcli.go

```diff
@@ -63,7 +63,7 @@ import (
 
 const (
 	// Version is the CLI version of buf.
-	Version = "1.10.0-dev"
+	Version = "1.11.2-dev"
```

Comment:
?? Overly aggressive search-and-replace?

Reply:
a bit of end to end manual testing gone rogue 😆 

have converted back to draft

Reply:
reverted in [731d00f](https://github.com/bufbuild/buf/pull/1628/commits/731d00f7abd8ddeb8d28c774669ac4ea6fa92d2d)